opal/mca/threads/pthreads/configure.m4

```diff
@@ -363,23 +364,15 @@ AC_DEFUN([OPAL_INTL_POSIX_THREADS_SPECIAL_FLAGS],[
 # -Kthread:
 # -kthread:  FreeBSD kernel threads
 # -pthread:  Modern GCC (most all platforms)
-# -pthreads: GCC on solaris
 # -mthreads:
-# -mt:       Solaris native compilers / HP-UX aCC
+# -mt:       HP-UX aCC
 #
 # Put -mt before -mthreads because HP-UX aCC will properly compile
 # with -mthreads (reading as -mt), but emit a warning about unknown
-# flags hreads.  Stupid compilers.
+# flags threads.  Stupid compilers.
```

Comment:
This was not a typo.  The compiler emitted a warning about unknown flags `hreads`, because the `t` was consumed as part of the `-mt` parsing.

Reply:
Done - I put quotes around the word so it is a little more obviously intentional.